packages/wrangler/templates/pages-template-plugin.ts

```diff
@@ -133,15 +133,19 @@ export default function (pluginArgs: unknown) {
 		const handlerIterator = executeRequest(request, relativePathname);
 		const pluginNext = async (input?: RequestInfo, init?: RequestInit) => {
 			if (input !== undefined) {
-				request = new Request(input, init);
+				let url = input;
```

Comment:
Was forgotten about when it was added to the OG template.

https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/templates/pages-template-worker.ts#L128-L140